### PR TITLE
Tidy up Gemfile and remove any unnecessary dependency

### PR DESCRIPTION
**Pull request by aliuk2012**
#### What problem does the pull request solve?
repeat of  https://github.com/alphagov/forms-runner/pull/362
Trello card:

#### Checklist

- [ ] I've used the pull request template
- [ ] I've linked this PR to the relevant issue (if mission work)
- [ ] I've written unit tests for these changes (if code change)
- [ ] I've updated the documentation in (If any documentation requires updating)
  - [ ] README.md
  - [ ] Elsewhere (please link)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -24,15 +24,10 @@ gem "warden"
...
 gem "pundit"
 
-# Build JSON APIs with ease [https://github.com/rails/jbuilder]
-gem "jbuilder"
-
 # Use Sentry (https://sentry.io/for/ruby/?platform=sentry.ruby.rails#)
 gem "sentry-rails"
 gem "sentry-ruby"
 
-gem "dotenv-rails", groups: %i[development test]
```
Comment (jamie-o-wilkinson) [inline Gemfile:34]: Could it be that some people use this locally for development? (I discovered I was by accident this week, had a .env file I had forgotten about and didn't realise was used! 😮 )

Comment (jamie-o-wilkinson) [reply Gemfile:34]: Oh I don't actually use it usually, I was wondering if anyone else did. I wanted to stash some env var values and threw them in an uncommitted .env file thinking I would just `source .env` if I needed them - little did I know that we used the dotenv gem and it tripped me up!

**Review by jamie-o-wilkinson — APPROVED** [review]
Nice to do some tidying up 👏

File: Gemfile (hunk 1 above)
Comment (thomasiles) [reply Gemfile:34]: I'm not using it

Comment (DavidBiddle) [reply Gemfile:34]: Me neither. I guess if anyone _is_ using this they can use `source .env` for this as Jamie says?

**Pull request merged**